### PR TITLE
Blank aria label fixes

### PR DESCRIPTION
**Pull request by austinoneil**
## Description

Fix issue where, when a blank aria label is passed in, the attribute would render on the inner elements for most elements.

References #2409

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Many, many e2e tests

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by coliff** [comment]
This looks great - thanks! Don't worry about the failed [accessibility-lint GitHub Action](https://github.com/dequelabs/axe-linter-action). This was something we recently added but it's a bit buggy and hasn't been updated in a while and has many deprecation warnings.

**Review by coliff — DISMISSED** [review]

File: stencil-workspace/src/components/modus-alert/modus-alert.tsx [hunk 1]
```diff
@@ -61,12 +61,20 @@ export class ModusAlert {
...
     const iconSize = '24';
     const MAX_LENGTH = 300;
 
+    const className = {
+      alert: true,
+      'with-action-button': (this.dismissible && this.buttonText?.length > 0) || this.buttonText?.length > 0,
+    };
+    const typeClass = this.classByType.get(this.type);
+    if (typeClass != undefined) {
+      className[typeClass] = typeClass;
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-alert/modus-alert.tsx:73]: Can you set `className[typeClass] = true`.  I feel this is working as typeClass is truthy, but `true` should avoid any confusion about what the value should be (true/false)

File: stencil-workspace/src/components/modus-alert/modus-alert.tsx [hunk 2]
```diff
@@ -61,12 +61,20 @@ export class ModusAlert {
...
+    }
+
     return (
-      <div aria-label={this.ariaLabel} class={className} role="alert">
+      <div aria-label={this.ariaLabel == '' ? null : this.ariaLabel} class={className} role="alert">
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-alert/modus-alert.tsx:77]: Later on you switch to `this.ariaLabel || undefined`, can we  update all locations to use the same optimized convention?

**Review by cjwinsor — CHANGES_REQUESTED** [review]

File: stencil-workspace/src/components/modus-alert/modus-alert.tsx (hunk 2 above)
Comment (austinoneil) [reply stencil-workspace/src/components/modus-alert/modus-alert.tsx]: Done.

File: stencil-workspace/src/components/modus-alert/modus-alert.tsx (hunk 1 above)
Comment (austinoneil) [reply stencil-workspace/src/components/modus-alert/modus-alert.tsx]: done

**Review by cjwinsor — APPROVED** [review]

**Pull request merged**